includes/Gateway/Cash_App_Pay_Gateway.php

```diff
@@ -1067,6 +1152,9 @@ protected function do_transaction( $order ) {
 				$location_id = $this->get_plugin()->get_settings_handler()->get_location_id();
 				$response    = $this->get_api()->create_order( $location_id, $order );
 
+				// TODO: save gift card order details
```

Comment:
Is this TODO still valid?

Reply:
This is no longer valid, as this is handled in #79 (The initial plan was to handle in this PR)